### ISSUE TITLE
Roboto font-weight 400 is not rendering properly

### ISSUE THREAD

**Issue by sathyamoorthi** [google/fonts#1568]
Hi,

We are using Roboto font for our most of the sites. Suddenly all of them have issues with the font. But only in chrome. Firefox rendering them fine. I tried developer.android.com as well where they are using Roboto font. There too i see the same issue. Can someone help to resolve it?

Screenshot is taken by rendering this page https://developer.android.com/topic/performance/vitals/frozen

![image](https://user-images.githubusercontent.com/1912654/40232423-52c32596-5abc-11e8-8c73-11f0c0853611.png)

**Comment by m4rc1e** [comment]
Roboto hasn't been updated this year. It was last modified 2017-10-16.

Which OS and Browser version are you using?

**Comment by sathyamoorthi** [comment]
Windows 10
Google Chrome Version 66.0.3359.139 (Official Build) (64-bit)

**Comment by m4rc1e** [comment]
Do you have Roboto installed locally?

**Comment by sathyamoorthi** [comment]
Yes. I had "Roboto Light" font installed in my windows. After I removed it, fonts are appearing good. Thanks for the insight. It is really helpful.

But this issue just happened today. I had that font for more than 2 years in my machine. Not sure what has updated in a day,

**Comment by zspitzer** [comment]
I have the same problem over at http://docs.lucee.org nothing has changed code wise, it changed in the last few days

**Comment by zspitzer** [comment]
same problem can be seen in https://web.whatsapp.com/ it's now barely legible

**Comment by zspitzer** [comment]
here is the chrome bug https://bugs.chromium.org/p/chromium/issues/detail?id=845259

**Comment by kgregory** [comment]
>Yes. I had "Roboto Light" font installed in my windows. After I removed it and cleared my browser cache, fonts are appearing good. Thanks for the insight. It is really helpful.

I've seen the same behavior and wanted to share my experience for anyone who may run into this:

My work laptop is a Dell and their SupportAssist utility installs Roboto fonts that Chrome will use.  On a few occasions, Dell's utility would update and install `Roboto Light`, which was a very lightweight version of Roboto that was being picked up by elements with a font-family of `Roboto`.  Whenever this happened, I removed the font from Windows and everything returned to normal.

Recently, SupportAssist was updated again and installed a font named `Roboto` that consisted of only italic variants.  With this font installed, elements with a font-family of `Roboto` were italicized in Chrome.  After removing that, everything returned to normal.

At the time of writing, this behavior only presents itself in Chrome.  Firefox and other browsers do not appear to be affected.  If your fonts are being rendered incorrectly in Chrome, check other browsers.  If they don't exhibit the same issue, check your locally installed fonts.  In Windows you can sort fonts by modification or creation date by viewing `%windir%/Fonts` in *Details* mode and adding the appropriate columns.

**Comment by AlbertRAR** [comment]
@kgregory Thank you! Indeed, the issue was caused by the latest SupportAssist update.

**Comment by pixelloop** [comment]
Thank you @kgregory this has been doing my head in all day after updating my Dell. All other advice didn't help, I deleted the font which sucks because I use it for design work but I can't stand everything in chrome being in italics.

**Comment by radiocity** [comment]
Loading fonts from browser (Google Chrome) cache cause this issue.

**Comment by radiocity** [comment]
@pixelloop, I`m using base64 woff fonts by fontsquirel with custom font-family name.
```scss
$font-primary: 'Roboto Web', 'Roboto', sans-serif;
$font-secondary: 'Ubuntu Web', 'Ubuntu', sans-serif;
```

**Comment by laerm0** [comment]
Hello all, it looks like your issues have been straightened out. Is anyone still having this issue?

**Comment by Guichaguri** [comment]
As @kgregory pointed out, the problem was indeed the SupportAssist update. Since I can't just uninstall the font (there are other Dell computers using the site), I had to move from Google Fonts to self-hosting the font with another name, unfortunately, as @radiocity suggested.

Is there a bug report for this issue in Chromium's bug tracker? Chrome should've downloaded the regular version since it wasn't available in the system.

**Comment by gloaec** [comment]
For people that land here having the same problem as me : 

You don't want to fetch the Roboto font from Google API, neither include it in your bundle, but you want this font to be used when people that have it installed on their device.
If so, you still have to declare the `@font-face` that will match the appropriate local files.

Here is an example for Roboto : 
https://gist.github.com/gloaec/82a22fe09c869e29b26a8ceb5cc83b97

Hope this helps. Cheers !
